libcodechecker/server/api/store_handler.py

```diff
@@ -5,6 +5,7 @@
 # -------------------------------------------------------------------------
 
 import base64
+from datetime import datetime
```

Comment:
I also do not see any usages of this import. Is this a fix for a crash?

Reply:
Please see my answer at the `os` import question.